app/models/alchemy/page/page_elements.rb

```diff
@@ -143,6 +143,16 @@ def element_definitions
       @_element_definitions ||= element_definitions_by_name(element_definition_names)
     end
 
+    # All element definitions defined for page's page layout including nestable element definitions
+    #
+    def descendent_element_definitions
+      definitions = element_definitions_by_name(element_definition_names)
+      definitions.select { |d| d.key?('nestable_elements') }.each do |d|
+        definitions += element_definitions_by_name(d['nestable_elements'])
+      end
+      definitions
```

Comment:
This can lead to duplicate element definitions, which we won't want. Please run `definitions.uniq { |d| d['name'] }.`